.gitlab/ornl/ascent.gitlab-ci.yml

```diff
@@ -102,6 +102,7 @@
     - rm -rf "$WORKDIR"
 
 Ascent Build:
+  needs: []
```

Comment:
I am struggling to understand the difference between dependencies and needs in gitlab terminology but other than my lack of knowledge these changes look reasonable. Great work. 

PS:
Dependencies just inherit artifacts form previous jobs.
Needs lets you run out of order of staging .
At least that my understanding.

Reply:
Your understanding is correct. I had to google to understand this. `needs: []` just lets a job run ASAP when the workflow starts. We probably only needs dependencies for reporting jobs on PNNL where partition matters

Reply:
I might just cherry-pick this fix into #20